data/korath/korath culture conversations.txt

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2024 by Arachi-Lover
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Korath Far'en Lai Prayer"
+	minor
+	invisible
+	to offer
+		random < 5
```

Comment:
Might be fun to attach this to a particular time, as if this were a recurring ritual.  Sets things up for possible future interactions.

```suggestion
		day >= 13
		day < 18
		random < 35
```